src/pages/portfolio/AccountOverviewSection.tsx

```diff
@@ -83,7 +91,7 @@ export const AccountOverviewSection = () => {
       id: 'open-positions',
       label: stringGetter({ key: STRING_KEYS.POSITION_MARGIN }),
       amount: mapIfPresent(equity?.current, freeCollateral?.current, (e, f) => e - f),
-      color: ColorToken.Green2,
+      color: ColorToken.Yellow1,
```

Comment:
would it be better to use the theme hook to access the color with it's semantic name? Just in case color theme changes the `yellow` or `green` hex in the future

Reply:
I think my goal here was to be the same color regardless of theme.